app/navigation/stacks/parentsOnboardingStackNavigator.tsx

```diff
@@ -18,8 +18,8 @@ export type ParentsOnboardingStackParams = {
   LanguageSelection: undefined;
   Acknowledgement: undefined;
   SignUpCredentials: undefined;
-  SignUpNickname: undefined;
-  SignUpAvatar: undefined;
+  SignUpNickname: { isChild: boolean };
+  SignUpAvatar: { isChild: boolean };
   CharmsIntroducing: undefined;
   OnboardingCarousel: undefined;
 };
```

Comment:
the same as above -- use predefined route params
https://stackoverflow.com/questions/60115818/initial-route-params-in-react-navigation-v5